src/common/abstractResume.ts

```diff
@@ -59,24 +64,28 @@ export abstract class ResumeCommand<T extends typeof SfCommand> extends SfComman
     const { flags } = await this.parse(this.constructor as Interfaces.Command.Class);
     // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
     this.flags = flags;
+    this.outputService = new OutputServiceFactory().forResume(this.operationType);
   }
 
   protected async resumeOperation(): Promise<PromotePipelineResult> {
-    const asyncJobId = this.flags['use-most-recent']
+    const asyncJobId: string = this.flags['use-most-recent']
       ? (await DeployPipelineCache.create()).getLatestKeyOrThrow()
       : (this.flags['job-id'] as string);
 
     // get the latest state of the async job and validate that it's resumable
     const doceOrg: Org = this.flags['devops-center-username'] as Org;
     let asyncJob: AsyncOperationResult = await fetchAsyncOperationResult(doceOrg.getConnection(), asyncJobId);
-    if (isNotResumable(asyncJob.sf_devops__Status__c)) {
+    if (asyncJob.sf_devops__Status__c && isNotResumable(asyncJob.sf_devops__Status__c)) {
       throw messages.createError('error.JobNotResumable', [asyncJobId, asyncJob.sf_devops__Status__c]);
     }
 
     // it is resumable so we can start monitoring the operation
-    this.log(`*** Resuming ${this.operationType} ***`);
-    this.log(`Job ID: ${bold(asyncJobId)}`);
-    const streamer: DoceMonitor = getAsyncOperationStreamer(doceOrg, this.flags.wait, asyncJobId);
+    this.outputService.setAorId(asyncJobId);
+
+    await this.outputService.printOpSummary();
+    this.outputService.printAorId();
```

Comment:
We don’t need to print the aorid before ?

Reply:
The output was like this before the PR. I don't know exactly.

Reply:
Before the PR we didn’t have a printSummary =P

Reply:
Yes, but we had the console.logs with that, I moved them inside the resume output service

Reply:
Ahhhh yes !!! My bad it's the resuming !!! I thought was the abstract promote class !!!
Sorry